### ISSUE TITLE
TS2304: Cannot find name 'DOMHighResTimeStamp'

### ISSUE THREAD

**Issue by timdream** [block65/webcrypto-web-push#15]
```
> tsc -p web-push-proxy/tsconfig.json

node_modules/@block65/webcrypto-web-push/dist/lib/types.d.ts:12:28 - error TS2304: Cannot find name 'DOMHighResTimeStamp'.

12     expirationTime: null | DOMHighResTimeStamp;
                              ~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@block65/webcrypto-web-push/dist/lib/types.d.ts:12
```

Thanks for making this library available! 

I setup my tsconfig to only import `worker` library (the project is intended to be executed as a CloudFlare Worker) and `DOMHighResTimeStamp` is not defined. This can simply be `number`.

**Comment by maxholman** [comment]
For posterity, I think this type error didn't show in dev due to the use of triple-slash directives in other files that essentially make those types available in the entire code base

**Comment by maxholman** [comment]
Published as v1.0.2
